### PR TITLE
allow logging in staging

### PR DESCRIPTION
**Pull request by douglaschan32167**
## Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. 

## Types of changes

- [x] I've labeled this PR with appropriate labels (`comp/*` component, bug/feature, documentation and other less important ones).

## Checklist
_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [x] I've linked the proper Github Issue(s) to my PR
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests (unit, integration...etc.) that prove my fix is effective or that my change works
- [x] I have added or updated necessary documentation (if appropriate). You can search, for instance using the [`silver searcher`](https://github.com/ggreer/the_silver_searcher) `ag <STRING>` to find all usages in the repository.
- [x] I have notified the team of any breaking or important changes

### CODE REVIEW

File: backend/api/settings.py [hunk 1]
```diff
@@ -143,6 +143,20 @@
...
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+    },
+    'root': {
+        'handlers': ['console'],
+        'level': 'DEBUG',
```
Comment (ktham) [inline backend/api/settings.py:156]: I think we should set this via env vars, maybe something like `os.getenv('LOG_LEVEL')`

File: backend/api/utility/views.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
 """Utility views that don't necessarily correspond to a model"""
 from django.http import HttpResponse
+import logging
 
+logger = logging.getLogger(__name__)
 
 def ping_view(request):
     """Should return 200 if hit. For the load balancer."""
+    logger.debug("health check")
```
Comment (ktham) [inline backend/api/utility/views.py:9]: This seems like it would help https://pypi.org/project/django-request-logging/#description

Comment (douglaschan32167) [reply backend/api/utility/views.py:9]: added. still keeping the manual logging statements though until we fix this.

File: backend/api/settings.py (hunk 1 above)
Comment (douglaschan32167) [reply backend/api/settings.py:156]: done

Comment (ktham) [reply backend/api/settings.py:156]: I still see it hardcoded as 'DEBUG'?

File: backend/api/utility/views.py (hunk 2 above)
Comment (ktham) [reply backend/api/utility/views.py:9]: Ok, just keep in mind that we pay for disk storage for logs too

**Pull request merged**